docs/interactive.md

```diff
@@ -0,0 +1,127 @@
+# Interactive Python Console
+
+Tomviz provides a Python console and interactive API that can be using to build up
+and interact with pipelines from within Python. The Python console can be enable
+using the View menu.
+
+
+# Load a state file
+
+```python
+>>>
+Python 3.7.5 (default, Nov 20 2019, 09:21:52)
+[GCC 9.2.1 20191008] on linux
+>>> from tomviz import state
+>>> state.load('/home/cjh/tmp/test.tvsm')
+>>>
+```
+
+# Executing the pipeline
+
+```python
+>>>
+Python 3.7.5 (default, Nov 20 2019, 09:21:52)
+[GCC 9.2.1 20191008] on linux
+>>> from tomviz import state
+>>> state.load('/home/cjh/tmp/test.tvsm')
+>>> state.pipelines[0].execute()
+>>>
+```
+
+# Creating a data source
+
+```python
+>>>
+Python 3.7.5 (default, Nov 20 2019, 09:21:52)
+[GCC 9.2.1 20191008] on linux
+>>> from tomviz import state
+>>> ds = state.DataSource(
+            fileNames=['/home/cjh/data/TestData3D/SMALL_PtCu_NanoParticle.tif']
+    )
+>>> p = state.Pipeline(ds)
+>>> state.pipelines.append(p)
+>>>
+```
+
+# Adding an operator
+
+```python
+>>>
+Python 3.7.5 (default, Nov 20 2019, 09:21:52)
+[GCC 9.2.1 20191008] on linux
+>>> from tomviz import state
+>>> ds = state.DataSource(
+        fileNames=['/home/cjh/data/TestData3D/SMALL_PtCu_NanoParticle.tif']
+    )
+>>> p = state.Pipeline(ds)
+>>> state.pipelines.append(p)
+>>> from tomviz.operators import InvertData
+>>> o = InvertData()
+>>> p.dataSource.operators.append(o)
+>>>
+```
+
+# Adding a Module
+
+```python
+>>>
+Python 3.7.5 (default, Nov 20 2019, 09:21:52)
+[GCC 9.2.1 20191008] on linux
+>>> from tomviz import state
+>>> ds = state.DataSource(
+        fileNames=['/home/cjh/data/TestData3D/SMALL_PtCu_NanoParticle.tif']
+    )
+>>> p = state.Pipeline(ds)
+>>> state.pipelines.append(p)
+>>> from tomviz.modules import Volume
+>>> m = Volume()
+>>> p.dataSource.modules.append(m)
+>>>
+```
+
+# Taking a screen shot
+
+```python
+>>>
+Python 3.7.5 (default, Nov 20 2019, 09:21:52)
+[GCC 9.2.1 20191008] on linux
+>>> from tomviz import state
+>>> state.load('/home/cjh/tmp/test.tvsm')
+>>> state.pipelines[0].execute()
+>>> from tomviz.views import active_view
+>>> view = active_view()
+>>> view.save_screenshot('/tmp/screen.png')
+```
+
+Controlling output image. The `palette`, `width` and `height` keyword arguments
```

Comment:
Should `Controlling output image` be a header, like the others? As in, `# Controlling output image`.